build.gradle

```diff
@@ -50,19 +47,15 @@ ext {
   fullVersion = baseVersion + ((!snapshotVersion && milestone) ? "-M$milestone" : "") + "-groovy-$variant" + (snapshotVersion ? "-SNAPSHOT" : '')
   variantLessVersion = baseVersion + (snapshotVersion ? "-SNAPSHOT" : (milestone ? "-M$milestone" : ""))
   javaVersions = [1.8, 11] // ensure that latest version is actually build on travis, otherwise no docs get published
-  javaVersion = System.getProperty("java.specification.version") as BigDecimal
-
-  if (javaVersion >= 9) {
-    groovyDependency += ["javax.xml.bind:jaxb-api:2.3.0"]
-  }
+  ext.javaVersion = System.getProperty("java.specification.version") as BigDecimal
```

Comment:
You are in an `ext` block, why again the `ext.`?

Reply:
Good point. That block is too long and I've missed it. I thought it was just a local variable. I will simplify it.